<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Xinkai Chen</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xware Desktop</string>
  </property>
  <property name="windowIcon">
   <iconset theme="xware-desktop"/>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CustomWebView" name="webView">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_createTask"/>
    <addaction name="action_refreshPage"/>
    <addaction name="separator"/>
    <addaction name="action_setting"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_showAbout"/>
   </widget>
   <widget class="QMenu" name="menu_backend">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>后端</string>
    </property>
    <addaction name="action_ETMstart"/>
    <addaction name="action_ETMstop"/>
    <addaction name="action_ETMrestart"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_backend"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="CustomStatusBar" name="statusBar_main">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar {
	background-color: skyblue;
}
::item {
	border: none !important;
}</string>
   </property>
  </widget>
  <action name="action_showAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_createTask">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>新建任务</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_setting">
   <property name="icon">
    <iconset theme="document-properties"/>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_refreshPage">
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>刷新页面</string>
   </property>
   <property name="shortcuts">
    <stringlist>
     <string>Ctrl+R</string>
     <string>F5</string>
    </stringlist>
   </property>
  </action>
  <action name="action_ETMstop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="action_ETMstart">
   <property name="text">
    <string>开启</string>
   </property>
  </action>
  <action name="action_ETMrestart">
   <property name="text">
    <string>重启</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>CustomWebView</class>
   <extends>QWebView</extends>
   <header>legacy/CustomWebView/CWebView</header>
  </customwidget>
  <customwidget>
   <class>CustomStatusBar</class>
   <extends>QStatusBar</extends>
   <header>legacy/CustomStatusBar/CStatusBar</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
